object_detection/README.md

```diff
@@ -1,95 +1,52 @@
-
-### Test data order
-Any order.
-
-# 4. Model
-### Publication/Attribution
-He, Kaiming, et al. "Mask r-cnn." Computer Vision (ICCV), 2017 IEEE International Conference on.
-IEEE, 2017.
-
-We use a version of Mask R-CNN with a ResNet50 backbone.
-
-### List of layers
-Running the timing script will display a list of layers.
-
-### Weight and bias initialization
-The ResNet50 base must be loaded from the provided weights. They may be quantized.
-
-### Loss function
-Multi-task loss (classification, box, mask). Described in the Mask R-CNN paper.
-
-Classification: Smooth L1 loss
-
-Box: Log loss for true class.
-
-Mask: per-pixel sigmoid, average binary cross-entropy loss.
-
-### Optimizer
-Momentum SGD. Weight decay of 0.0001, momentum of 0.9.
-
-# 5. Quality
-### Quality metric
-As Mask R-CNN can provide both boxes and masks, we evaluate on both box and mask mAP.
-
-### Quality target
-Box mAP of 0.377, mask mAP of 0.339
-
-### Evaluation frequency
-Once per epoch, 118k.
-
-### Evaluation thoroughness
-Evaluate over the entire validation set. Use the official COCO API to compute mAP.
+# Benchmark execution with MLCube
+
+## Current implementation
+
+We'll be updating this section as we merge MLCube PRs and make new MLCube releases.
+
+### Project setup
+```Python
+# Create Python environment and install MLCube Docker runner 
+virtualenv -p python3 ./env && source ./env/bin/activate && pip install mlcube-docker
+
+# Fetch the Object Detection workload
+git clone https://github.com/mlcommons/training && cd ./training
+git fetch origin pull/501/head:feature/object_detection && git checkout feature/object_detection
+cd ./object_detection/mlcube
+```
+
+### Dataset
+
+
+The COCO dataset will be downloaded and extracted. Sizes of the dataset in each step:
+
+| Dataset Step                   | MLCube Task       | Format         | Size     |
+|--------------------------------|-------------------|----------------|----------|
+| Download (Compressed dataset)  | download_data     | Tar/Zip files  | ~20.5 GB |
+| Extract (Uncompressed dataset) | download_data     | Jpg/Json files | ~21.2 GB |
+| Total                          | (After all tasks) | All            | ~41.7 GB |
+
+### Tasks execution
+
+Parameters are defined at these files:
+
+* MLCube user parameters: mlcube/workspace/parameters.yaml
+* Project user parameters: pytorch/configs/e2e_mask_rcnn_R_50_FPN_1x.yaml
+* Project default parameters: pytorch/maskrcnn_benchmark/config/defaults.py
+
+```
+
+# Download COCO dataset. Default path = /workspace/data
+python mlcube_cli.py run --task download_data --platform docker
+
+# Run benchmark. Default paths = ./workspace/data
+python mlcube_cli.py run --task train --platform docker
+```
+
+Parameters defined at **mculbe/mlcube.yaml** could be overridden using: `--param=input`
+
+We are targeting pull-type installation, so MLCube images should be available on docker hub. If not, try this:
+
+```bash
+mlcube run ... -Pdocker.build_strategy=always
+```
```

Comment:
[This](https://github.com/mlcommons/training/blob/master/benchmark_readme_template.md) is the benchmark README template. So can you please add sections that have been moved to README.old back?

Reply:
Done, thanks for pointing this out, move mlcube explanation into the mlcube folder